presto-main/src/main/java/io/prestosql/server/DataSizeSerializer.java

```diff
@@ -0,0 +1,32 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.server;
+
+import com.fasterxml.jackson.core.JsonGenerator;
+import com.fasterxml.jackson.databind.JsonSerializer;
+import com.fasterxml.jackson.databind.SerializerProvider;
+import io.airlift.units.DataSize;
+
+import java.io.IOException;
+
+public class DataSizeSerializer
+        extends JsonSerializer<DataSize>
+{
+    @Override
+    public void serialize(DataSize dataSize, JsonGenerator jsonGenerator, SerializerProvider serializerProvider)
```

Comment:
do we still need `toBytesValueString()` in `DataSize`?

Reply:
Let's ask @martint and @electrum as they discussed this change in https://github.com/airlift/units/pull/16.

Also @martint left comment:

```
I think we should externalize the serialization to JSON. Having both toByteValueString() and toString() is confusing, and the former exists solely for the purpose of serializing it to be used in JSON.
```

Which is exactly what this PR does.